### DISCUSSION TITLE
Please - Duplicates that prioritise HEIC format to keep

### DISCUSSION THREAD

**Discussion by vw-kombi** [immich-app/immich#12964] [General]
Another new version - and I eagerly await something many iPhone users would like - duplicates that prioritise HEIC format.
This is the only thing stopping me going forward with this.
Just a flag in settings - Proritise HEIC in duplicates, and then the HEIC format is the one to be kept.
20,000+ dupes too much to manually do.
Each new version I go to the dupes and see which is to be 'saved' and always the inferior jpg file that is over twice the size!

**Comment by MicHi07i** [comment]
I also look forward to some rule sets, like prioritaizing format, quality, size, foldernames. Due to merging some backups I have lots of duplicates. I'd like to save time, avoiding hours of manually wlking through the vast majority of my merged library.
